### PR TITLE
Modify the max vcpu number to adapt to the ppc default xml

### PR DESCRIPTION
**Pull request by Grace-chhu**
Signed-off-by: Chenli Hu <chhu@redhat.com>

### CODE REVIEW

**Comment by Grace-chhu** [comment]
# avocado run --vt-type libvirt virsh.guestvcpus
......
TESTS      : 8
 (1/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.normal_test.no-option: PASS (43.78 s)
 (2/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.normal_test.disable: PASS (58.36 s)
 (3/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.normal_test.enable: PASS (56.31 s)
 (4/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.normal_test.combine: PASS (57.24 s)
 (5/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.error_test.invalid_dom.no_dom: PASS (54.44 s)
 (6/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.error_test.invalid_dom.err_dom: PASS (54.41 s)
 (7/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.error_test.invalid_cpulist.enable: PASS (54.47 s)
 (8/8) type_specific.io-github-autotest-libvirt.virsh.guestvcpus.error_test.invalid_cpulist.disable: PASS (55.81 s)
RESULTS    : PASS 8 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_guestvcpus.cfg [hunk 1]
```diff
@@ -6,7 +6,7 @@
     encode_video_files = "no"
     skip_image_processing = "yes"
     take_regular_screendumps="no"
-    vcpus_num = "4"
+    vcpus_num = "20"
```
Comment (chunfuwen) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_guestvcpus.cfg]: Is there any reason why need increase vcpus_num value?

Comment (Grace-chhu) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_guestvcpus.cfg]: Hi, chunfuwen

The customer default xml use so many vcpus. Thank you!

**Comment by chunfuwen** [comment]
minor mistake I made here, I just merge this PR  before I click approve button.

**Pull request merged**